softdev.bib

```diff
@@ -688,6 +712,23 @@ @phdthesis{visser97sdf
     school = {University of Amsterdam}
 }
 
+@book{visser97scannerless,
```

Comment:
Indentation is wrong. And I think we have this already (and it's a phdthesis)?

Reply:
Something is wrong with this entry. Is it a technical report? If so, it should have a number. Is it a PhD thesis? If so, we already have an entry claiming to be Eelco's PhD thesis, so at most one of those can be correct.

Reply:
Sorry, didn't see this. This seems to be a report. I'll update it.